posit-dev/positron
# python: do not refresh interpreters on quickpick open

related to #3944 

Keeps the lists of interpreters the same, rather than refreshing only the first time the Select Interpreter QuickPick is opened which makes it seem like the Positron dropdown is out of sync. 

This nudges people towards refreshing manually when they have created a new environment, which I believe to be the desired behavior since Positron will not have to continually watch for new envs (which can be expensive, some discussion https://github.com/posit-dev/positron/issues/2003). I've opened #4269 to track better UI to make refreshing interpreters more obvious.

### QA Notes



Scenario 1:

1. Open Positron
2. Create new pyenv environment via something like `pyenv virtualenv 3.9.4 test-startup`
3. Check: no new env discovered in Positron dropdown, no new env discovered in `Python: Select Interpreter`
4. Refresh envs using button in `Python: Select Interpreter`
5. See env populate in both locations

Scenario 2:

1. Open Positron
2. Create new pyenv environment via something like `pyenv virtualenv 3.9.4 test-startup`
3. Close Positron
4. Open Positron
5. See env populate in both locations
